DocumentDB/examples/secured_setup/main.tf

```diff
@@ -0,0 +1,25 @@
+# A more secure example of a DocumentDB setup
+module "test_secured_documentdb" {
+  source                 = "../../"
+  enable                 = true
+  database_name          = "secured_documentdb"
+  billing_code           = "billing_code_test"
+  master_username        = "test"
+  master_password        = "another_test"
+  subnet_ids             = module.subnets.private_subnet_id
+  vpc_security_group_ids = [module.security_group-documentdb.security_group_ids]
+  storage_encrypted      = true
+  kms_key_id             = module.kms_key.key_arn
```

Comment:
Same comment here if you could add these resources just to show their expected values.

Reply:
@patheard - I changed the example, so let me know if it is sufficient.